app/helpers/hosts_helper.rb

```diff
@@ -517,4 +517,19 @@ def power_status_visible?
   def host_breadcrumb
     breadcrumbs(resource_url: "/api/v2/hosts?thin=true'")
   end
+
+  def remediation_help_text
+    case @host.build_status_label
+      when "Pending installation"
+        N_("Install media and if build fails, rebuild again")
+      when "Token expired"
+        N_("Cancel your build and regenerate token by initiating build again")
+      when "Installed"
+        N_("Done!")
+      when "Installation error"
+        N_("Check audit log")
```

Comment:
Probably: `OS installer post script reported failure, check logs` (I don't think this can be found in audit but in normal log)